src/app/component/profile/pages/my-account-edit/my-account-edit.component.ts

```diff
@@ -0,0 +1,117 @@
+import { Component, OnDestroy, OnInit } from "@angular/core";
+import { List } from "immutable";
+import { Subject } from "rxjs";
+import { takeUntil } from "rxjs/operators";
+import { PageComponent } from "src/app/helpers/page/pageComponent";
+import { Page } from "src/app/helpers/page/pageDecorator";
+import { AnyMenuItem } from "src/app/interfaces/menusInterfaces";
+import { User } from "src/app/models/User";
+import { APIErrorDetails } from "src/app/services/baw-api/api.interceptor";
+import { UserService } from "src/app/services/baw-api/user.service";
+import {
+  editMyAccountMenuItem,
+  myAccountCategory,
+  myAccountMenuItem
+} from "../../profile.menus";
+import data from "./my-account-edit.json";
+
+@Page({
+  category: myAccountCategory,
+  menus: {
+    actions: List<AnyMenuItem>([myAccountMenuItem, editMyAccountMenuItem]),
+    links: List()
+  },
+  self: editMyAccountMenuItem
+})
+@Component({
+  selector: "app-my-account-edit",
+  template: `
+    <app-wip>
+      <app-form
+        *ngIf="ready"
+        [schema]="schema"
+        [title]="'Profile Settings'"
+        [error]="errorEdit"
+        [success]="successEdit"
+        [submitLabel]="'Update'"
+        [submitLoading]="loading"
+        [btnColor]="'btn-warning'"
+        (onSubmit)="submitEdit($event)"
+      ></app-form>
+
+      <app-form
+        *ngIf="ready"
+        [schema]="{ model: {}, fields: [] }"
+        [title]="'Cancel my account'"
+        [subTitle]="'Unhappy? You can permanently cancel your account.'"
+        [error]="errorDelete"
+        [success]="successDelete"
+        [submitLabel]="'Cancel my account'"
+        [submitLoading]="loading"
+        [btnColor]="'btn-danger'"
+        (onSubmit)="submitDelete($event)"
+      >
+      </app-form>
```

Comment:
Error and success values should only go through the top form, otherwise alerts will popup out of place.